docs/source/overview/connect-to-data-sources.rst

```diff
@@ -6,6 +6,12 @@ EvaDB supports a wide range of data sources including SQL database systems, obje
 Connect to an Existing SQL Database System
 ------------------------------------------
 
+.. note::
+
+   Connecting to an existing SQL database is unnecessary for inserting or querying data in Eva. This step is only required if
+   users intend to work with data already present in some SQL DB. Eva has native storage support for SQLite, which can be utilized without
+   establishing any connection. Refer to :ref:`EvaQL<evaql>` page for more details.
```

Comment:
Hi, I think the idea is that we want the user to use the third party data sources instead of the native storage support for SQLite.  Notice the third party data source also has SQLite support, which does not require an existing SQL database. We have also updated the documentation page of each third party data source to indicate whether it requires an already running database instance. This [page](https://evadb.readthedocs.io/en/stable/source/reference/databases/postgres.html) is an example for  PostgreSQL.

Reply:
Thanks @Chitti-Ankith for the feedback! We can add a note in [Connect to an Existing SQL Database System](https://evadb.readthedocs.io/en/stable/source/overview/connect-to-data-sources.html) that says `For local prototyping, we can establish a connection to [sqlite](https://evadb.readthedocs.io/en/stable/source/reference/databases/sqlite.html)`. I think the key is that we want the user to establish a connection (e.g., to SQLite) instead of using the native storage support.

Reply:
Done